kernel/configs/fragments/common/huge.conf

```diff
@@ -1,5 +1,10 @@
 # Items to enable large/huge mmu pages and tlbs etc.
 
+# Compaction is the only memory management component to form high order
+# (larger physically contiguous) memory blocks reliably. The lack of the
+# feature can lead to unexpected OOM killer invocations for high order memory requests.
+CONFIG_COMPACTION=y
```

Comment:
ooh, nice catch - yes, having checked, we probably do want this enabled.
Note to others, this is a config change for x86.

Reply:
Nice catch @Pennyzct